content/html/concepts/tags/terms/audio/audio.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '<audio>'
+Description: 'Represents an interface for adding audio content to the page.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Represents an interface for adding audio content to the page.
+
+HTML supports the following audio formats:
+
+- MP3
+- WAV
+- OGG
+
+## Syntax
+
+```html
+<audio controls>Source tags with different audio formats go here</audio>
+```
+
+The `<audio>` tag may contain one or more `<source>` tags with different audio formats. The browser will run the format most suitable for the browser.
+
+The `<audio>` tag may include `<source>` tags and fallback text. It may also use attributes to convey intention, some common ones are below:
```

Comment:
```suggestion
The `<audio>` tag may include `<source>` tags and fallback text. It may also use attributes to convey intention, such as the ones listed below:
```